app/desktop/studio_server/eval_api.py

```
@@ -846,3 +874,164 @@ async def get_eval_configs_score_summary(
             partially_rated_count=partially_rated_count,
             not_rated_count=not_rated_count,
         )
+
+    @app.get(
+        "/api/projects/{project_id}/tasks/{task_id}/run_config/{run_config_id}/eval_scores"
+    )
+    async def get_run_config_eval_scores(
```

Pylint (3.3.7)

[refactor] Too many local variables (41/15)

(R0914)